README.md

```diff
@@ -1,3 +1,17 @@
+<p align="center">
+  <a href="https://rubyonrails.org/" target="_blank" rel="noopener noreferrer">
+    <img src="https://rubyonrails.org/images/rails-logo.svg" width="400">
+  </a>
+</p>
+
+<p align="center">
```

Comment:
Can you remove all those badges? They are already in the contributing.md file given them are more relevant to contributors than users and the README file is targeted to users of the framework.

Reply:
Ah, they are not. Can you move them to there?

Reply:
I can but I think it's cool for new comers to have a quick overview of these Rails "stats"

Reply:
I understand, but those stats don't add much value or are already are in the page:

* master should be always green and make no different for new comers
* The number of downloads should not matter, those download are inflated by CI systems anyway
* The number of star is on the top of the repository page
* Let last version of the framework is on the sidebar on the Releases topic
* The License is also in the sidebar

Reply:
You're right, will change